README.md

```diff
@@ -1,3 +1,38 @@
+![Path18 logo](logo.png)
 
-# Path18 
-Web App for exposing young people to a variety of career paths :mortar_board:
+##### Web App for exposing young people to a variety of career paths :mortar_board:
```

Comment:
maybe it is better to say
web app for informing young people of a variety of careers and paths?